src/library/scala/math/Ordering.scala

```diff
@@ -422,10 +422,11 @@ object Ordering extends LowPriorityOrderingImplicits {
     }
     implicit object IeeeOrdering extends IeeeOrdering
   }
-  @deprecated("There are multiple ways to order Floats (Ordering.Float.TotalOrdering, " +
-    "Ordering.Float.IeeeOrdering). Specify one by using a local import, assigning an implicit val, or passing it " +
-    "explicitly. See their documentation for details.", since = "2.13.0")
-  implicit object DeprecatedFloatOrdering extends Float.TotalOrdering
+  implicit object DeprecatedFloatOrdering extends Ordering[Float] {
```

Comment:
If you do something like this it might be nice to add a comment explaining why it's got the word `Deprecated` while not being deprecated.

Reply:
Probably... :(

Reply:
Or call it 
```
object `**********FloatOrdering`
```